### DISCUSSION TITLE
LYWSD03MMC B1.6 expected battery life for Zigbee Firmware Z03MMC_v0117

### DISCUSSION THREAD

**Discussion by MikealSouth** [pvvx/ZigbeeTLc#40] [Q&A] [answered]
Hi,
I am running a bunch of LYWSD03MMC B1.6 thermometers via Zigbee using the Z03MMC_v0117 firmware, and just after two weeks most batteries are already dead.
Is that expected?
As I understand it the devices are sending the data by themselves and Z2M doesn't poll them. How can I set a much less infrequent transmits rate?
What am I doing wrong?
Thanks,
Mike
<img width="571" alt="Screenshot 2024-01-01 at 12 16 43" src="https://github.com/pvvx/ZigbeeTLc/assets/153289734/495927af-d13d-4739-a08e-d55404a369ff">

**Comment by pvvx** [comment, marked as answer]
> How can I set a much less infrequent transmits rate?

There are standard functions for this in Zigbee 3.0.
Change the minimum and maximum reporting interval, and delta parameters for each clusters: for temperature, humidity and battery.

**Reply by MikealSouth** [reply]
I just googled that, makes perfect sense,I learned something, thank you.
How much battery life should I expect?

**Reply by pvvx** [reply]
[For B1.6 the reviews are still bad](https://github.com/pvvx/ZigbeeTLc/issues/37). The reasons for the rapid drainage of the battery have not been established. Not enough information...
